src/conn/conn_open.c

```diff
@@ -204,7 +204,7 @@ __wt_connection_workers(WT_SESSION_IMPL *session, const char *cfg[])
     /*
      * Run recovery.
      * NOTE: This call will start (and stop) eviction if recovery is
-     * required.  Recovery must run before the history store table is created
+     * required. Recovery must run before the history store table is created
      * (because recovery will update the metadata), and before eviction is
      * started for real.
      */
```

Comment:
I suggest doing `git checkout -- conn_open.c` and leave this out of your change.

Reply:
Fixed